Desktop

tingamapuro04 opened the pull request.

tingamapuro04 committed:
Styled first and second container in the desktop version of my portflio project

tingamapuro04 committed:
Added images files for the desktop

tingamapuro04 committed:
Styling of the third container desktop version

tingamapuro04 committed:
Added css styling for desktop view

tingamapuro04 committed:
Linter corrections

tingamapuro04 committed:
Linter corrections

tingamapuro04 committed:
Linter corrections

tingamapuro04 commented:
I was working on the Desktop branch where I was trying to apply styling to the mobile version so that i can also have a desktop version of the page. My break point is set at 768pixels where the switch between mobile and desktop takes place. I have a main container enclosing the four other containers inside. Inside the first container I have just a mere introduction and a background cover image.
in the second container I have four cards displayed with flex and flex direction is column with things done .
In the third container I have Information about self with  display flex and flex direction row. This container spans the whole width.
In the last container I have a contact form with action method of "post".

youmari commented on index.html:
```diff
@@ -17,15 +17,25 @@
+
+                <nav>
+                    <div class="hidden">
+                        <ul class="hidden-ul">
+                            <li class="po"><a href="#">Portoflio</a></li>
+                            <li class="ab"><a href="#">About</a></li>
+                            <li class="co"><a href="#">Contact</a></li>
+                        </ul>
```
your navigation links should  have the same style as the ones in the Figma design, so please it would be better if you try to make the same Figma design 
the font ,color and text decoration

- [ ] Yours
![image](https://user-images.githubusercontent.com/61790587/167229349-5ae30797-55b6-49ec-b5be-c31a49591389.png)

- [ ] Figma design 
![image](https://user-images.githubusercontent.com/61790587/167229373-8e5596bf-0dc8-4935-8cc5-33c6befd474d.png)

youmari commented on index.html:
```diff
@@ -17,15 +17,25 @@
     <div class="Main-container">
         <div class="container">
             <header class="header hoverable">
-                <a>My Logo</a>
-                <nav><img src="/photos/Union.png" alt="Hamburger Icon"></nav>
+                <h1 class="gol"><a>My Logo</a></h1>
+
+                <nav>
+                    <div class="hidden">
+                        <ul class="hidden-ul">
+                            <li class="po"><a href="#">Portoflio</a></li>
+                            <li class="ab"><a href="#">About</a></li>
+                            <li class="co"><a href="#">Contact</a></li>
+                        </ul>
+                    </div>
```
- [ ] Your portfolio should be responsive in all devices but I noticed that  there is a gap space in 670px width screens, so it would be better if you fix it in order your portfolio become responsive in all devices
![image](https://user-images.githubusercontent.com/61790587/167229494-7a72cc11-3649-4dce-bf79-68c97f37dd3b.png)

youmari commented on index.html:
```diff
@@ -17,15 +17,25 @@
         <div class="container">
             <header class="header hoverable">
-                <a>My Logo</a>
-                <nav><img src="/photos/Union.png" alt="Hamburger Icon"></nav>
```
- [ ] your Images doesn't show up because of case sensitive of your src link it would be better if you try to add a dot before the slash like that `src="./photos/Union.png"` that will make your image show up 
and also do that to all your images link

youmari commented on index.html:
```diff
@@ -37,8 +47,8 @@ <h1> <strong>I’m Mariane
                     <ul class="media">
                         <li> <a href="https://twitter.com/alphonce_mobutu"><img class="twi" src="/photos/twitter.jpg" alt="Twitter Icon"> </a></li>
-                        <li> <a href="https://www.linkedin.com/in/adoyo-alphonce-3362a4173/?lipi=urn%3Ali%3Apage%3Ad_flagship3_feed%3BkY43KWq1SIegum1Uq5M6dA%3D%3D"><img class="lin" src="/photos/Linkedin-icon.jpg" alt="Linkedin Icon"></a></li>
-                        <li> <a href="https://www.facebook.com/me/"><img class="mil" src="/photos/mail.jpg" alt="Mil Icon"> </a></li>
+                        <li><a href="https://www.linkedin.com/in/adoyo-alphonce-3362a4173/?lipi=urn%3Ali%3Apage%3Ad_flagship3_feed%3BkY43KWq1SIegum1Uq5M6dA%3D%3D"><img class="lin" src="/photos/Linkedin-icon.jpg" alt="Linkedin Icon"></a></li>
+                        <li><a href="https://www.facebook.com/me/"><img class="mil" src="/photos/mail.jpg" alt="Mil Icon"></a></li>
                         <li><a href="https://github.com/tingamapuro04"><img class="git" src="/photos/github.jpg" alt="github Icon"> </a></li>
                         <li> <a href="https://github.com/tingamapuro04"><img class="angel" src="/photos/angellist.jpg" alt="Angellist Icon"></a></li>
```
- [ ] please try to change the src link as I suggest for more details check the comment above

youmari commented on index.html:
```diff
@@ -17,15 +17,25 @@
     <div class="Main-container">
         <div class="container">
             <header class="header hoverable">
```
- [ ] please it would be better if you remove that line in the top 
![image](https://user-images.githubusercontent.com/61790587/167230121-0a0ff466-6abe-4563-b4d7-f894cdd733b0.png)

youmari reviewed (changes requested):
Hi @tingamapuro04 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ No linter error
✔️ Great short summary

## Required Changes ♻️

- [ ] I suggest changing your projects buttons to the ones in the Figma design to do that you should change the color border and the background color of the button 

- [ ] Yours
![image](https://user-images.githubusercontent.com/61790587/167229859-70c0bacc-3094-48c3-8ef3-9680575ccf31.png)

- [ ] Figma design 
![image](https://user-images.githubusercontent.com/61790587/167229884-70abb44c-0b2f-429c-8234-3dcb2a5cfa6f.png)

- [ ] Your contact section doesn't look as the one in the Figma design please I suggest sticking to the Figma design 
![image](https://user-images.githubusercontent.com/61790587/167229968-2fd09c89-f186-41d5-bd06-44c827eb8dc3.png)
![image](https://user-images.githubusercontent.com/61790587/167230004-0345316d-487e-4857-be52-600672a9c5db.png)



_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

tingamapuro04 committed:
Added styling for median query min-width:375px

tingamapuro04 committed:
Linter corrections

tingamapuro04 committed:
Linter corrections

tingamapuro04 committed:
Added styling to button

tingamapuro04 committed:
Added styling to button

tingamapuro04 commented:
I was doing changes on the desktop branch. I have done the following
1. I have added a media query at min-width=375px
2.  I have enabled hover and active properties of the button and applied the background color and color as per figma design
3.  I have removed the text decoration from the nav bar
4. I have added a . to all image src

tingamapuro04 committed:
background size correction

AmaduKamara commented on index.html:
```diff
@@ -205,43 +232,45 @@ <h1>About
-                <footer class="submit">
-                    <button type="submit">Get my resume</button>
-                </footer>
+                <div class="submit resume">
+                    <button  type="submit">Get my resume</button>
+                </div>
```
- [ Optional Changes ] I see that when you were referencing the Figma design, there was a border around the button. That was because other students or viewers were hovering on that button and Figma made that border appear, but it's not meant to be in the original design. Please remove the border on the div that wraps the button to make your design look great 👍 

**Your Design**
![image](https://user-images.githubusercontent.com/50941074/167250585-833fa745-1297-494d-9bd5-b3ebb1391d71.png)

**You see, there is no border wrapping the button here on the Figma design**
![image](https://user-images.githubusercontent.com/50941074/167250620-f3745bfc-4086-4f7e-9dde-e08f46356513.png)

AmaduKamara reviewed (approved):
# Status: Approved 🟢🟢🟢 

Hi @tingamapuro04,

![YoureDoingAGoodJobGreatJobGIF](https://user-images.githubusercontent.com/50941074/167250983-3ff03f7b-8a05-4192-8ac7-c5cf9b2cc212.gif)

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

## Optional Changes 

- [ ] Kindly change this gray border on the buttons for the mobile view to a blue border as in the Figma design.

![image](https://user-images.githubusercontent.com/50941074/167250730-d9e1e8ef-3963-4f8a-9114-2b6e12b04599.png)

**The Figma design**
![image](https://user-images.githubusercontent.com/50941074/167250779-15478254-b3bf-4d82-a71b-447af223e609.png)

- [ ] It would be nice if you remove this dotted border on the text area and try to match the Figma desig.
- Your design
![image](https://user-images.githubusercontent.com/50941074/167250850-2debd94a-d4c7-42b2-88e5-c71fb5ff5221.png)

- The Figma design has no border on the textarea
![image](https://user-images.githubusercontent.com/50941074/167250897-6449cced-3771-4819-a44b-37347d4b47c5.png)


- [ ] To complete it all, please remove the big space below the contact section to the end of the page. After the blue background of the contact section, there is no space anymore. That is the end of the page. 👍 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```
